src/clr-angular/forms/datepicker/date-container.spec.ts

```diff
@@ -94,6 +98,16 @@ export default function() {
 
                 sub.unsubscribe();
             });
+
+            // IE doesn't support MouseEvent constructors
+            itIgnore(["ie"], "marks the date control as touched when the datepicker popover is toggled", () => {
+                spyOn(dateFormControlService, "markAsTouched");
+
+                const fakeEvent: MouseEvent = new MouseEvent("fakeEvent");
+                context.clarityDirective.toggleDatepicker(fakeEvent);
```

Comment:
I don't believe you need the MouseEvent. You could literally pass `null` and the test would still work.